### ISSUE TITLE
An unexpected error has occurred

### ISSUE THREAD

**Issue by chrisguindon** [EclipseFdn/react-eclipsefdn-members#197]
Perri and Zahara, both got "An unexpected error has occurred" when they submitted their form.

Martin did some initial research and found that their internal organization_id was missing in our database.

I tried to reproduce the error with Zahara on a call but it worked. I am starting to wonder if this error only occurs for new users?

**Comment by autumnfound** [comment]
There was no row for organization in the table for their given form ID, so it likely there was a failure when posting that was ignored. I don't know the case where this happens, so we would need a reproduction case to accurately fix this

**Comment by linkfang** [comment]
A quick way to re-produce the issue.

- create a new branch (like, test) from master.
- make sure the db is empty and restart the docker image.
- open rem.docker website, sign in and go ahead with the new form application.

The POST request is good, like so:
![image](https://user-images.githubusercontent.com/43075439/126372061-4f6f9cd1-9b72-41a5-aa26-b7c90c3a20ff.png)
![image](https://user-images.githubusercontent.com/43075439/126372069-55912037-b38f-4380-a132-84a2f3e2f253.png)
![image](https://user-images.githubusercontent.com/43075439/126372079-cfe81bbe-5b62-435d-aee7-8567d418d61a.png)

But if we refresh the page and choose "continue with existing", the GET request for address and organization would give us an empty array. And there is no org and address data in db as well.
![image](https://user-images.githubusercontent.com/43075439/126372225-82ae7a9b-d2fd-4cef-9f27-eaca43879fb1.png)
![image](https://user-images.githubusercontent.com/43075439/126372233-6816b3fc-073d-4096-a166-c83c6b43982e.png)

**Comment by autumnfound** [comment]
So, I managed to get this to trigger once then never again, even after trying to do it again from the beginning. I'm thinking this is some sort of weird race condition, and my initial build was in a funky state and restarting got us in an OK state. With the current master branch, I am unable to reproduce this with any reliability. It might have been a side-effect of fixing the weird deletion/update issue we had, and that it was caused by the same thing.
